### ISSUE TITLE
st2chatops never fires an action

### ISSUE THREAD

**Issue by manasdk** [StackStorm/st2chatops#38]
From user report on support@stackstorm.com or googlegroups.

"This was, in fact, a problem with the matching rules. All of the sample 
commands I had found used !rules and !help - this did not work. Everything 
immediately worked fine when I used '! rules' and '! help'. Hope this helps 
someone along the way."

This sounds like a bug with out help string geenration?

**Comment by emedvedev** [comment]
The only case I know where Hubot would respond to `! help` but wouldn't respond to `!help` is explicitly setting the Hubot alias to `!` (with a space). This is a very strange thing to do and not the case with our default env: https://github.com/StackStorm/st2chatops/blob/master/st2chatops.env#L17
With `!` it will respond to both `!help` and `! help`. I just confirmed that this is the case in Slack (that person used Slack as well). 

Basically, that person's Hubot (not any of our code and not `hubot-help`) broke in a strange way. I haven't seen other reports of this happening, and I think it might be an env configuration issue. I couldn't reproduce this behavior, so I'm going to wait for a few days for any kind of similar reports, and then close the issue.

**Comment by emedvedev** [comment]
Something else that's on my mind is that we stopped bundling `!rules` (which is used as an example in the issue description) a long time ago. It's very likely it just was an outdated manual / article of some sort.
